apache-ansible-demo/docs/prerequisites.md

```diff
@@ -0,0 +1,9 @@
+To run this resource package, you require the following prerequisites:
```

Comment:
Could this sentence be phrased better?
For example
To run the bash scripts and execute the example adopt procedure contained in this resource package, you require the following prerequisites:

Reply:
I'm going to rephrase this, but based on the following considerations:
```
Those are the prerequisites to run the resource package in ALM. Not only for the adopt example but in general if you want to use the resource package to create, delete start and stop the network service through ALM.

The prerequisites in the adopt example have a link to them as they apply to it too.

About the bash scripts, the scripts.md should articulate the requirements for them separately.
```